advanced-javascript/arrays.spec.js

```diff
@@ -40,15 +40,27 @@ describe('Arrays', () => {
     });
   });
 
+  describe('find', () => {
+    it('should be a function', () => {
+      const find = arrayFunctions.find;
+      expect(typeof find).toBe('function');
+    });
+    it('should return the odd numbers in the array', () => {
+      const find = arrayFunctions.find;
+      const arr = [6, 7, 2];
+      expect(find(arr, num => num % 2 === 0)).toEqual([6])
+    });
+  });
+
   describe('filter', () => {
     it('should be a function', () => {
       const filter = arrayFunctions.filter;
       expect(typeof filter).toBe('function');
     });
     it('should return the odd numbers in the array', () => {
       const filter = arrayFunctions.filter;
-      const arr = [6,7,2]
-      expect(filter(arr, (num => num < 4)).toEqual([2])
+      const arr = [6,7,2];
+      expect(filter(arr, num => num % 2 === 1)).toEqual([7])
```

Comment:
when we have so many parentheses it is easy to mess up the function. 
Try something like
let result = ```filter(arr, num => num % 2 === 1)``` 
```expect(result).toEqual([7])```